OloEditor/src/Panels/PostProcessSettingsPanel.cpp

```
@@ -15,6 +17,13 @@ namespace OloEngine
 
         ImGui::Begin("Post Processing");
 
+        // Snapshot before any UI for undo tracking
+        auto& settings = Renderer3D::GetPostProcessSettings();
+        if (m_CommandHistory && !m_IsEditing)
```

Clang (14.0.6)

[warning] implicit conversion 'OloEngine::CommandHistory *' -> bool

(readability-implicit-bool-conversion)